Makefile

```diff
@@ -173,7 +173,8 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 build-installer: manifests generate kustomize ## Generate a consolidated YAML with CRDs and deployment.
 	mkdir -p dist
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
-	$(KUSTOMIZE) build config/default > dist/install.yaml
+	$(KUSTOMIZE) build config/default > dist/cloudflare-operator.yaml
+	$(KUSTOMIZE) build config/crd > dist/cloudflare-operator.crds.yaml
```

Comment:
I think the config/default also includes the CRDs?

Reply:
The cert-manager release includes CRDs and a general installer (https://github.com/cert-manager/cert-manager/releases/tag/v1.17.2). Managing CRDs and the operator separately could be beneficial.